Issue releasing v0.18.0 to Maven Central

```
[INFO] Waiting until Deployment 7a381e96-68ce-4e57-b98e-798c6b505921 is published
Error:  

Deployment 7a381e96-68ce-4e57-b98e-798c6b505921 failed
pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem@0.18.0:
 - Project name is missing

pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem-quarkus@0.18.0:
 - Project name is missing

pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem-validator@0.18.0:
 - Project name is missing

pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem-java-ee@0.18.0:
 - Project name is missing

pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem-bom@0.18.0:
 - Project name is missing

pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem-spring-boot-2@0.18.0:
 - Project name is missing

pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem-spring-boot-3@0.18.0:
 - Project name is missing

pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem-quarkus-deployment@0.18.0:
 - Project name is missing

pkg:maven/io.github.belgif.rest.problem/belgif-rest-problem-spring@0.18.0:
 - Project name is missing


[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for io.github.belgif.rest.problem:belgif-rest-problem-parent 0.18.0:
[INFO] 
[INFO] io.github.belgif.rest.problem:belgif-rest-problem-parent SUCCESS [ 14.824 s]
[INFO] belgif-rest-problem ................................ SUCCESS [  8.770 s]
[INFO] belgif-rest-problem-validator ...................... SUCCESS [  3.750 s]
[INFO] belgif-rest-problem-java-ee ........................ SUCCESS [  8.005 s]
[INFO] belgif-rest-problem-spring ......................... SUCCESS [  5.111 s]
[INFO] belgif-rest-problem-spring-boot-2 .................. SUCCESS [  3.812 s]
[INFO] belgif-rest-problem-spring-boot-3 .................. SUCCESS [  4.614 s]
[INFO] belgif-rest-problem-quarkus ........................ SUCCESS [  9.249 s]
[INFO] belgif-rest-problem-quarkus-deployment ............. SUCCESS [  3.581 s]
[INFO] belgif-rest-problem-it ............................. SUCCESS [  0.240 s]
[INFO] belgif-rest-problem-apt ............................ SUCCESS [  1.398 s]
[INFO] belgif-rest-problem-it-common ...................... SUCCESS [  2.264 s]
[INFO] belgif-rest-problem-spring-boot-2-it ............... SUCCESS [ 20.262 s]
[INFO] belgif-rest-problem-spring-boot-3-it ............... SUCCESS [ 19.748 s]
[INFO] belgif-rest-problem-java-ee-it ..................... SUCCESS [ 54.047 s]
[INFO] belgif-rest-problem-jakarta-ee-it .................. SUCCESS [ 32.803 s]
[INFO] belgif-rest-problem-quarkus-it ..................... SUCCESS [02:54 min]
[INFO] belgif-rest-problem-jackson-minimal-it ............. SUCCESS [  1.429 s]
[INFO] belgif-rest-problem-jackson-latest-it .............. SUCCESS [  1.381 s]
[INFO] belgif-rest-problem-codegen-it ..................... SUCCESS [  3.476 s]
[INFO] belgif-rest-problem-bom ............................ FAILURE [ 11.296 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  06:26 min
[INFO] Finished at: 2025-11-27T14:50:28Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.sonatype.central:central-publishing-maven-plugin:0.9.0:publish (injected-central-publishing) on project belgif-rest-problem-bom: Execution injected-central-publishing of goal org.sonatype.central:central-publishing-maven-plugin:0.9.0:publish failed: Deployment 7a381e96-68ce-4e57-b98e-798c6b505921 (belgif-rest-problem) failed while publishing -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :belgif-rest-problem-bom
Error: Process completed with exit code 1.
```

Looks like we maybe need to fill in `<name>` in the pom.xml of all submodules?

--- a/belgif-rest-problem-bom/pom.xml
+++ b/belgif-rest-problem-bom/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-bom</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>pom</packaging>
   <description>Maven BOM (Bill of Materials) for dependency versions of belgif-rest-problem modules</description>
 

--- a/belgif-rest-problem-it/belgif-rest-problem-apt/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-apt/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-apt</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
 
   <dependencies>
     <dependency>

--- a/belgif-rest-problem-it/belgif-rest-problem-codegen-it/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-codegen-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-codegen-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
 
   <build>
     <plugins>

--- a/belgif-rest-problem-it/belgif-rest-problem-it-common/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-it-common/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-it-common</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
 
   <dependencies>
     <dependency>

--- a/belgif-rest-problem-it/belgif-rest-problem-jackson-latest-it/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-jackson-latest-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-jackson-latest-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
 
   <dependencies>
     <dependency>

--- a/belgif-rest-problem-it/belgif-rest-problem-jackson-minimal-it/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-jackson-minimal-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-jackson-minimal-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
 
   <dependencies>
     <dependency>

--- a/belgif-rest-problem-it/belgif-rest-problem-jakarta-ee-it/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-jakarta-ee-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-jakarta-ee-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>war</packaging>
 
   <build>

--- a/belgif-rest-problem-it/belgif-rest-problem-java-ee-it/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-java-ee-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-java-ee-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>war</packaging>
 
   <properties>

--- a/belgif-rest-problem-it/belgif-rest-problem-quarkus-it/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-quarkus-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-quarkus-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
 
   <dependencyManagement>

--- a/belgif-rest-problem-it/belgif-rest-problem-spring-boot-2-it/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-spring-boot-2-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-spring-boot-2-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>war</packaging>
 
   <dependencyManagement>

--- a/belgif-rest-problem-it/belgif-rest-problem-spring-boot-3-it/pom.xml
+++ b/belgif-rest-problem-it/belgif-rest-problem-spring-boot-3-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-spring-boot-3-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>war</packaging>
 
   <dependencyManagement>

--- a/belgif-rest-problem-it/pom.xml
+++ b/belgif-rest-problem-it/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-it</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>pom</packaging>
   <description>integration tests</description>
 

--- a/belgif-rest-problem-java-ee/pom.xml
+++ b/belgif-rest-problem-java-ee/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-java-ee</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
 
   <properties>

--- a/belgif-rest-problem-quarkus-deployment/pom.xml
+++ b/belgif-rest-problem-quarkus-deployment/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-quarkus-deployment</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
 
   <dependencyManagement>

--- a/belgif-rest-problem-quarkus/pom.xml
+++ b/belgif-rest-problem-quarkus/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-quarkus</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
 
   <dependencyManagement>

--- a/belgif-rest-problem-spring-boot-2/pom.xml
+++ b/belgif-rest-problem-spring-boot-2/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-spring-boot-2</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
 
   <dependencyManagement>

--- a/belgif-rest-problem-spring-boot-3/pom.xml
+++ b/belgif-rest-problem-spring-boot-3/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-spring-boot-3</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
 
   <dependencyManagement>

--- a/belgif-rest-problem-spring/pom.xml
+++ b/belgif-rest-problem-spring/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-spring</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
 
   <dependencyManagement>

--- a/belgif-rest-problem-validator/pom.xml
+++ b/belgif-rest-problem-validator/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem-validator</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
   <description>Validation library that generates Belgif-compliant BadRequestProblem with InputValidationIssues
     (https://www.belgif.be/specification/rest/api-guide/#bad-request).

--- a/belgif-rest-problem/pom.xml
+++ b/belgif-rest-problem/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>belgif-rest-problem</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>jar</packaging>
   <description>Support classes for generating standardized problem types
     (https://www.belgif.be/specification/rest/api-guide/#standardized-problem-types).

--- a/jacoco-aggregator/pom.xml
+++ b/jacoco-aggregator/pom.xml
@@ -10,6 +10,7 @@
   </parent>
 
   <artifactId>jacoco-aggregator</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
   <packaging>pom</packaging>
 
   <dependencies>

